test/es-module/test-esm-loader.mjs

```diff
@@ -0,0 +1,73 @@
+// Flags: --experimental-loader ./test/fixtures/es-module-loaders/hooks-custom.mjs
+import '../common';
+import assert from 'assert';
+
+
+assert.rejects(
```

Comment:
All those needs to be `await`ed otherwise we won't know if there's a never settling promise.
```suggestion
await assert.rejects(
```